build/index_core_compatible.js

```diff
@@ -1,20 +1,37 @@
-import Experiment from '../es6/experiment';
-import Interpreter from '../es6/interpreter';
-import RandomPlanoutCoreCompatible from '../es6/ops/randomPlanoutCoreCompatible';
-import Core from '../es6/ops/core';
-import * as Namespace from '../es6/namespace';
 import Assignment from '../es6/assignment';
+import Experiment from '../es6/experiment';
 import ExperimentSetup from '../es6/experimentSetup';
+import Interpreter from '../es6/interpreter';
+import * as LibUtils from '../es6/lib/utils';
+import Random from '../es6/ops/randomPlanoutCoreCompatible';
+import * as Base from '../es6/ops/base';
+import * as Core from '../es6/ops/core';
+import * as OpsUtils from '../es6/ops/utils';
+import * as Namespace from '../es6/namespacePlanoutCoreCompatible';
+
+OpsUtils.registerOperators({
+    "randomFloat": Random.RandomFloat,
+    "randomInteger": Random.RandomInteger,
+    "bernoulliTrial": Random.BernoulliTrial,
+    "bernoulliFilter": Random.BernoulliFilter,
+    "uniformChoice": Random.UniformChoice,
+    "weightedChoice": Random.WeightedChoice,
+    "sample": Random.Sample
+}, true);
```

Comment:
@jcwilson I'm going to block off some time 2 weeks from today to heavily restructure planout.js for a v4.0 release.

Are you cool with sticking to 2.0.2 until then? I'm worried that we'd need to bump the major version if we merge this PR, which might result in people opting into a very temporary version of the project. Though I might be over thinking things a bit 😅